### PR TITLE
1969/update cookie package

### PR DESCRIPTION
**Pull request by Ma9sha**
@derekjohnson, can you Review the changes. I would also like to test it out locally if it is possible.

### CODE REVIEW

File: consent-banner.es5.js [hunk 1]
```diff
@@ -33,25 +34,54 @@ window.BarnardosConsent = function(options) {
...
     buttonWrap.appendChild(rejectButton);
     buttonWrap.appendChild(acceptButton);
     consentBanner.appendChild(buttonWrap);
+    consentBanner.appendChild(closeButton);
     // Put first in the DOM so keyboard and AT users can interact with it quickly
     var { firstChild } = document.body;
     firstChild.parentNode.insertBefore(consentBanner, firstChild);
+    consentBanner.parentNode.insertBefore(cookieOverlay, consentBanner);
+    // Get the focusable elements and focus the cookie notice
+    const focusableElements = consentBanner.querySelectorAll("a, button");
```
Comment (derekjohnson) [inline consent-banner.es5.js:73]: We can't use `const` in this file, it's not ES5. Use `var` instead.

File: consent-banner.es5.js [hunk 2]
```diff
@@ -70,9 +100,9 @@ window.BarnardosConsent = function(options) {
       'gtm.start': new Date().getTime(),
       event: 'gtm.js'
     });
-    var f = d.getElementsByTagName(s)[0],
-      j = d.createElement(s),
-      dl = l != 'dataLayer' ? '&l=' + l : '';
+    var f = d.getElementsByTagName(s)[0];
+    var j = d.createElement(s);
+    var dl = l != 'dataLayer' ? '&l=' + l : '';
```
Comment (derekjohnson) [inline consent-banner.es5.js:105]: This is a good change 👍

File: consent-banner.es5.js [hunk 3]
```diff
@@ -184,5 +228,37 @@ window.BarnardosConsent = function(options) {
       sendClickAction(e.target);
     });
   }
-};
 
+  if (cookieOverlay) {
+    cookieOverlay.addEventListener("click", e => {
```
Comment (derekjohnson) [inline consent-banner.es5.js:233]: Can't use arrow functions in ES5. This (and the next few instances) need to be changed to
```js
cookieOverlay.addEventListener("click", function(e) (
```

**Review by derekjohnson — CHANGES_REQUESTED** [review]

**Comment by derekjohnson** [comment]
@Ma9sha due to scoping in JS the vars declared inside the `buildBanner()` function are `undefined` outside it. Because of this you need to move the declarations for `cookieOverlay` and `closeButton` outside the function like this:
```js
  var consentBanner = document.createElement('div');
  var rejectButton = buildButton('Reject');
  var acceptButton = buildButton('Accept');
  var cookieOverlay = document.createElement("div");
  var closeButton = document.createElement("button");
```

This is the same for all 3 files.

**Comment by Ma9sha** [comment]
@derekjohnson , Thanks for the PR, updated the changes and its ready for another glance

**Review by derekjohnson — APPROVED** [review]

**Pull request merged**